app/models/post.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: posts
+#
+#  id         :bigint           not null, primary key
+#  body       :text             not null
+#  status     :integer          default("public"), not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  user_id    :bigint
+#
+# Indexes
+#
+#  index_posts_on_user_id  (user_id)
+#
+class Post < ApplicationRecord
+  belongs_to :user
+  has_one :picture, as: :picturable
```

Comment:
It'll be nice to create an additional PR with renaming `picturable` to `pictureable` to follow convention :)

Reply:
Ok, but this needs to be done later:)